src/static/js/overlay.js

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
❓ I don't understand why the js and html files have been renamed. We manage overlays in general here.
Is it for consistency with something else? I don't see what.

Reply:
I deleted this file to have only one HTML page with the rendering-diagram. But as we had a lot of unvisible buttons, the screenshot of the tests was bigger. So I split again the 2 pages.

Reply:
reverted